src/DataMiner/SLManagedAutomation/UIBuilder.cs

```diff
@@ -280,6 +280,31 @@ public class UIBuilder
 		/// </example>
 		public UIBuilder AppendButton(string destVar, string displayText) { return null; }
 
+
+		/// <summary>
+		/// Adds a button to this dialog box with the specified destination variable name, button text and style.
+		/// The supported button styles can be accessed through const strings on the Style.Button class.
+		/// </summary>
+		/// <param name="destVar">The name of the destination variable.</param>
+		/// <param name="displayText">The button text.</param>
+		/// <param name="style">The button style (see Style.Button for supported styled).</param>
```

Comment:
```suggestion
		/// <param name="style">The button style (see Style.Button for supported styles).</param>
```

Reply:
As an extra comment, should we make a separate section/page for the available styles. Now they can't be found in the Help (i.e. that you have None, Title1, Title2 and Title3 for the text styles and None and CallToAction for the button styles)